From 9be06cd4783f921f9fd8032a5157f0cca5f275c3 Mon Sep 17 00:00:00 2001
From: d34d <clark@cyngn.com>
Date: Wed, 9 Dec 2015 10:38:18 -0800
Subject: [PATCH 14/80] LockPatternUtils: Make settings getter and setters
 protected

This patch will allow us to extend LockPatternUtils in lineage-sdk
and have access to getting and setting lock settings, and minimize
code changes to the original LockPatternUtils class.

Change-Id: I3417f64db1f05cffab47328df574f23dd20776d5
---
 .../android/internal/widget/LockPatternUtils.java    | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/core/java/com/android/internal/widget/LockPatternUtils.java b/core/java/com/android/internal/widget/LockPatternUtils.java
index 7c339fb6d6b..7757b6e93dd 100644
--- a/core/java/com/android/internal/widget/LockPatternUtils.java
+++ b/core/java/com/android/internal/widget/LockPatternUtils.java
@@ -1330,7 +1330,7 @@ public class LockPatternUtils {
         return deadline;
     }
 
-    private boolean getBoolean(String secureSettingKey, boolean defaultValue, int userId) {
+    protected boolean getBoolean(String secureSettingKey, boolean defaultValue, int userId) {
         try {
             return getLockSettings().getBoolean(secureSettingKey, defaultValue, userId);
         } catch (RemoteException re) {
@@ -1338,7 +1338,7 @@ public class LockPatternUtils {
         }
     }
 
-    private void setBoolean(String secureSettingKey, boolean enabled, int userId) {
+    protected void setBoolean(String secureSettingKey, boolean enabled, int userId) {
         try {
             getLockSettings().setBoolean(secureSettingKey, enabled, userId);
         } catch (RemoteException re) {
@@ -1347,7 +1347,7 @@ public class LockPatternUtils {
         }
     }
 
-    private long getLong(String secureSettingKey, long defaultValue, int userHandle) {
+    protected long getLong(String secureSettingKey, long defaultValue, int userHandle) {
         try {
             return getLockSettings().getLong(secureSettingKey, defaultValue, userHandle);
         } catch (RemoteException re) {
@@ -1355,7 +1355,7 @@ public class LockPatternUtils {
         }
     }
 
-    private void setLong(String secureSettingKey, long value, int userHandle) {
+    protected void setLong(String secureSettingKey, long value, int userHandle) {
         try {
             getLockSettings().setLong(secureSettingKey, value, userHandle);
         } catch (RemoteException re) {
@@ -1364,7 +1364,7 @@ public class LockPatternUtils {
         }
     }
 
-    private String getString(String secureSettingKey, int userHandle) {
+    protected String getString(String secureSettingKey, int userHandle) {
         try {
             return getLockSettings().getString(secureSettingKey, null, userHandle);
         } catch (RemoteException re) {
@@ -1372,7 +1372,7 @@ public class LockPatternUtils {
         }
     }
 
-    private void setString(String secureSettingKey, String value, int userHandle) {
+    protected void setString(String secureSettingKey, String value, int userHandle) {
         try {
             getLockSettings().setString(secureSettingKey, value, userHandle);
         } catch (RemoteException re) {
-- 
2.17.1

